src/main/java/com/paneedah/mwc/network/messages/RealisticSoundMessage.java

```diff
@@ -0,0 +1,52 @@
+package com.paneedah.mwc.network.messages;
+
+import io.netty.buffer.ByteBuf;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.SoundEvent;
+import net.minecraft.util.math.BlockPos;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessage;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class RealisticSoundMessage implements IMessage {
+    boolean silencer;
+    private SoundEvent sound;
+    private BlockPos pos;
+    private int dimensionId;
```

Comment:
my weird obsession with ordering again
```suggestion
    private boolean silencer;
    
    private int dimensionId;
    
    private SoundEvent sound;
    private BlockPos pos;
```